src/components/StokesAnalysis/StokesAnalysisComponent.tsx

```diff
@@ -139,6 +139,20 @@ export class StokesAnalysisComponent extends React.Component<WidgetProps> {
         this.widgetStore.clearScatterPlotXYBounds();
     };
 
+    // true: red->blue, false: blue->red
+    private getColorMapOrder(frame: FrameStore): boolean {    
+        if (frame && frame.channelInfo && frame.channelInfo.fromWCS && frame.channelInfo.channelType && !isNaN(frame.channelInfo.delta)) {
+            let CTYPE = frame.spectralType !== null? frame.spectralType : frame.channelInfo.channelType.code;
+            if (CTYPE === SpectralType.CHANNEL) {
+                CTYPE = frame.channelInfo.channelType.code;
```

Comment:
this code is confusing to follow. if ctype is channel, then why do we go looking for the channel info's spectral type?

Reply:
when user select `Channel` in `conversion`, we use the default colour order from the file according default file spectral type.